### PR TITLE
Add Status page monitoring

### PR DESCRIPTION
**Pull request by matofeder**
Closes https://github.com/SovereignCloudStack/issues/issues/656

### CODE REVIEW

File: status-page/status-page-values.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          revision: 1
+          datasource:
+            - name: DS_PROMETHEUS
+              value: thanos
+        status-page-go:
+          gnetId: 10826
+          revision: 1
+          datasource:
+            - name: DS_PROMETHEUS
+              value: thanos
```
Comment (joshuai96) [inline status-page/status-page-values.yaml:73]: The [go metrics dashboard](https://grafana.com/grafana/dashboards/10826-go-metrics/) is designed to be used in k8s. Is it possible, using the relabeling from `values-workload.yaml`, to add namespace and podname to the metrics, to use the selectors on the top?

This is a nice to have and is non-blocking.

File: status-page/status-page-values.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        status-page-api:
+            url: https://raw.githubusercontent.com/SovereignCloudStack/status-page-deployment/17a247819dba1f1150889616ba3931b904f9419b/kubernetes/environments/kind/monitoring/grafana/dashboards/API-1718710757996.json
+        status-page-dex:
+            url: https://raw.githubusercontent.com/SovereignCloudStack/status-page-deployment/17a247819dba1f1150889616ba3931b904f9419b/kubernetes/environments/kind/monitoring/grafana/dashboards/Dex-1718271569643.json
+        status-page-oathkeeper:
+            url: https://raw.githubusercontent.com/SovereignCloudStack/status-page-deployment/17a247819dba1f1150889616ba3931b904f9419b/kubernetes/environments/kind/monitoring/grafana/dashboards/Oathkeeper-1718203458929.json
```
Comment (joshuai96) [inline status-page/status-page-values.yaml:79]: I'll open a new PR, when i come around changing the dashboards. This can be merged imho.

**Review by joshuai96 — APPROVED** [review]
LGTM

File: status-page/status-page-values.yaml (hunk 2 above)
Comment (joshuai96) [reply status-page/status-page-values.yaml:79]: If https://github.com/SovereignCloudStack/status-page-deployment/pull/11 gets merged first, I'll add the dashboards here.

**Review by chess-knight — APPROVED** [review]

**Pull request merged**